core/src/test/java/org/apache/iceberg/TestManifestEncryption.java

```diff
@@ -85,7 +85,7 @@ public class TestManifestEncryption {
 
   private static final DataFile DATA_FILE =
       new GenericDataFile(
-          0,
+          SPEC.specId(),
```

Comment:
?

Reply:
completing the previous patch https://github.com/apache/iceberg/pull/8252#discussion_r1523684641